src/fragments/lib/datastore/flutter/sync/10-installPlugin.mdx

```diff
@@ -21,11 +23,20 @@ import 'amplifyconfiguration.dart';
 import 'models/ModelProvider.dart';
 ```
 
+Then update your configure function by adding `AmplifyAPI` like mentioned. On earlier implementation configuration was used with `_configureAmplify` function and it can be updated there:
```

Comment:
nit: can we specify the "function" name here? I think we are super specific about that throughout the getting started guide @salihgueler

Reply:
I already mention it actually but this comment means it is not clear enough, let me rephrase myself :)